content/community/list of third party apps.de.md

```diff
@@ -28,6 +28,7 @@ ebenfalls nur quelloffene Projekte zu verlinken.
 
 | Projekt      | Plattform | Link(s)                                                                                                                                         | Status     | Infos                                                                                                                    |
 |--------------|-----------|-------------------------------------------------------------------------------------------------------------------------------------------------|------------|--------------------------------------------------------------------------------------------------------------------------|
+| TräwelCheckIn | iOS   | [https://twitter.com/TraewelCheckIn](https://twitter.com/TraewelCheckIn) | Private Beta | Träwel CheckIn ist eine iOS-App, welche das Einchecken und Interagieren mit Träwelling nativ unter iOS ermöglicht. |
```

Comment:
Can you add a link to the source code of the app?

Reply:
I believe the code is not public yet hence the beta, let me get back to you once I have clarification from the developer. I think the app was supposed to be open source anyways